src/gConsent/util/access.mock.ts

```diff
@@ -37,6 +37,7 @@ import type { AccessGrant } from "../type/AccessGrant";
 import type { AccessRequest } from "../type/AccessRequest";
 
 type RequestVcOptions = Partial<{
+  issuer: string;
```

Comment:
That's good, but you'll want this to have an effect on the resulting VC when calling `mockAccessRequestVcObject`. See how the VC object takes the provided options, or defaults to some arbitrary values if no option is passed in.